### ISSUE TITLE
Insert Title Here

### ISSUE THREAD

**Issue by ganiba** [WolframResearch/Chatbook#491]
Describe the issue in detail here. Attach any relevant screenshots or files. The section below was automatically generated. Remove any information that you do not wish to include in the report.

<details>
<summary>Debug Data</summary>

| Property | Value |
| --- | --- |
| Name | ``"Wolfram/Chatbook"`` |
| Version | ``"1.0.4"`` |
| EvaluationEnvironment | ``"Session"`` |
| FrontEndVersion | ``"Desktop: 13.3 for Mac OS X ARM (64-bit) (July 27, 2023)"`` |
| KernelVersion | ``"13.3.1 for Mac OS X ARM (64-bit) (July 27, 2023)"`` |
| SystemID | ``"MacOSX-ARM64"`` |
| Notebooks | ``True`` |
| DynamicEvaluation | ``False`` |
| SynchronousEvaluation | ``False`` |
| TaskEvaluation | ``False`` |

## Settings

| Property | Value |
| --- | --- |
| Assistance | ``True`` |
| LLMEvaluator | ``"CodeAssistant"`` |

## Stack Data
```
catchTop
throwFailure
trimURL
```

## Failure Data

```
<|
  "Evaluation" :> Wolfram`Chatbook`Actions`Private`writeChunk[Dynamic[Wolfram`Chatbook`Actions`Private`container$82354], CellObject["dcbb3a0f-e622-44ee-b4d9-f624d8ad54a8", "0992704d-979b-4f8e-bb98-cdac0a75a060"], "\":null,\"choices\":[{\"index\":0,\"delta\":{\"role\":\"assistant\",\"content\":\"\"},\"finish_reason\":null}]}\n\n"]
  "Arguments" -> {}
|>
```

</details>

**Comment by raverydavis** [comment]
This may have been resolved in a more recent version please try updating and report back if you still see the same issue. You can update the paclet using `PacletInstall["Wolfram/Chatbook", UpdatePacletSites -> True]`

**Comment by rhennigan** [comment]
This was fixed in #342.

Evaluate the following to get the latest updates:
```
PacletInstall["Wolfram/Chatbook", UpdatePacletSites -> True]
```
